### PR TITLE
0.0.1

### LINKED ISSUE #6: Remove messages that say `labeled the ...`
They're too much for our chat.

### PR DESCRIPTION
**Pull request by sadasant**
:) first approach so far

Fixes #6

### CODE REVIEW

File: index.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	"github.com/go-telegram-bot-api/telegram-bot-api"
+	"gopkg.in/go-playground/webhooks.v5/github"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+// IMPORTANT:
+// I tried to separate this in several files, but Zeit didn't let me
```
Comment (sadasant) [inline index.go]: Zeit seems pretty limited for Go code. But the server should work. Let's see.

Comment (sadasant) [reply index.go]: This is a test.

Comment (sadasant) [reply index.go]: This is another test.

**Comment by sadasant** [comment]
THIS IS WORKING NOW

**Review by mredvard — APPROVED** [review]

**Comment by guerrerocarlos** [comment]
Testing emojis 🚁🥁🎫🏙

**Review by ghostbar — APPROVED** [review]
lgtm

**Review by betacar — APPROVED** [review]

**Review by stefanmaric — APPROVED** [review]

**Pull request merged**